h/auth/__init__.py

```diff
@@ -6,7 +6,6 @@
 
 from pyramid.authentication import RemoteUserAuthenticationPolicy
 import pyramid_authsanity
-from pyramid_multiauth import MultiAuthenticationPolicy
```

Comment:
Indeed it does. I've removed the dependency in https://github.com/hypothesis/h/pull/4353/commits/3bdc8ec22e13330ffbe6e2afc83de9780ea2fe8b